### PR TITLE
JS UI test  for flow back navigation

### PR DESCRIPTION
**Pull request by bartzet**
# Problem
Test for flow back navigation is in Ruby.

# Solution
Implement flow back navigation test in JS. 
Remove Ruby UI test.

## Checklist
_put `n/a` if item is not relevant to PR changes_

- [ ] Have the CHANGELOG, README, MIGRATION, RELEASE_GUIDELINES docs been updated?
- [ ] Have new automated tests been implemented?
- [ ] Have new manual tests been written down?
- [ ] Have tests passed locally?
- [ ] Have any new strings been translated?

### CODE REVIEW

File: test/js/pageobjects/Welcome.js [hunk 1]
```diff
@@ -44,6 +44,14 @@ class Welcome extends Base {
   async closeModal() {
     this.closeButton.click()
   }
+
+  async backArrowNotDisplayed() {
```
Comment (kopijunkie) [inline test/js/pageobjects/Welcome.js]: I think the naming here is a bit misleading. A function name should say what it does, e.g. `checkBackArrowIsDisplayed`

Comment (bartzet) [reply test/js/pageobjects/Welcome.js]: I'm expecting it not to be displayed hence I catch the error. Selenium doesn't provide the check to verify whether the element is not displayed.

Improved the name of the function 561ae1f

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://740-pr-onfido-sdk-ui-onfido.surge.sh

File: test/js/pageobjects/Welcome.js [hunk 2]
```diff
@@ -44,6 +45,14 @@ class Welcome extends Base {
   async clickOnCloseModalButton() {
     this.closeModalButton.click()
   }
+
+  async checkBackArrowIsNotDisplayed() {
```
Comment (stefaniacardenas) [inline test/js/pageobjects/Welcome.js]: We should have this method also on the Complete screen

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-740-pr-onfido-sdk-ui-onfido.surge.sh

File: test/js/pageobjects/Welcome.js (hunk 2 above)
Comment (bartzet) [reply test/js/pageobjects/Welcome.js]: Added this to one of the tests in 78f828f

**Comment by rfreitas** [comment]
Travis automatic deployment: https://staging-740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://740-pr-onfido-sdk-ui-onfido.surge.sh

**Comment by rfreitas** [comment]
Travis automatic deployment: https://release-740-pr-onfido-sdk-ui-onfido.surge.sh

**Pull request merged**